### PR TITLE
UI changes for addition of new tab (serverless)

### PR DESCRIPTION
**Pull request by shantanu73**
1) UI changes for addition of new tab (serverless).

### CODE REVIEW

File: tests/js/spec/components/platformIconTile.spec.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    const PLATFORM_COLORS = {javscript: ['#3060b8']};
+    expect(selectPlatfrom(PLATFORM_COLORS, platforms)).toEqual(['#3060b8']);
+  });
+  it('should return appropriate style from getColorStyles for a platform when provided with object of colors', function() {
+    const platforms = ['javscript'];
+    const arg = {monoTone: '', platforms};
+    expect(getColorStyles(arg).styles).toEqual('background-color:#625471;color:#fff;');
+  });
+  it('should return appropriate JSX', function() {
+    expect(PlatformIconTile).toMatchSnapshot();
```
Comment (EvanPurkhiser) [inline tests/js/spec/components/platformIconTile.spec.js:19]: @billyvg are we still using snapshots?

File: src/sentry/static/sentry/app/data/platformCategories.tsx [hunk 2]
```diff
@@ -93,14 +103,17 @@ export const sourceMaps: PlatformKey[] = [
   'electron',
 ];
 
-export const tracing = ['python-tracing', 'node-tracing'] as const;
+// TODO(epurkhiser): I've added these since there was a need to have them in
+// the platform key type. However I have not added them anywhere else.
```
Comment (EvanPurkhiser) [inline src/sentry/static/sentry/app/data/platformCategories.tsx:107]: Did I write this comment? Where did it come from 🤔

File: tests/js/spec/components/platformIconTile.spec.js (hunk 1 above)
Comment (billyvg) [reply tests/js/spec/components/platformIconTile.spec.js:19]: Yeah I'll need to write a eslint plugin for this, but let's use `toSnapshot()`

File: src/sentry/static/sentry/app/data/platformCategories.tsx (hunk 2 above)
Comment (ajjindal) [reply src/sentry/static/sentry/app/data/platformCategories.tsx:107]: lol

File: src/sentry/static/sentry/app/data/platformCategories.tsx [hunk 3]
```diff
@@ -93,14 +103,17 @@ export const sourceMaps: PlatformKey[] = [
...
+const tracing = ['python-tracing', 'node-tracing'] as const;
```
Comment (EvanPurkhiser) [inline src/sentry/static/sentry/app/data/platformCategories.tsx:108]: Why isn't this exported anymore?

**Comment by EvanPurkhiser** [comment]
https://github.com/getsentry/sentry/pull/20723 superseded by

**Pull request closed without merging**